Documentation/Signoffs/Mounting.md

```diff
@@ -0,0 +1,123 @@
+
+This is in order to make sure the system is small and compact and does impede movement throughout the classroom.
+
+#### 4.) System shall have a mount or multiple mounts so that it can be installed in the classroom in an unfrequented area. [Origin: System Requirements]
+
+This will allow the system to perform its functionality without causing difficulty for students or faculty performing
+their required tasks.
+
+#### 5.) System shall be easy to remove and setup in a different room. [Origin: Broader Impacts]
+
+Other classrooms will experience noise and the ease in which the system can be moved is a large factor for how quickly noise in other rooms can be effectively canceled.
+
+#### 6.)  System shall minimize its visual distraction [Origin: Stakeholder Survey]
+
+Classrooms should be distraction free to allow students to learn new information effectively, therefore the system will not be distracting to students or faculty.
+
+#### 7.) System shall follow OSHA standard 1910.95 for occupational noise exposure. [Origin: OSHA standards]
+
+The mounting subsystem should be designed to ensure that the weighted average noise does not exceed 85 dB from the overall system to guarantee no hearing loss will occur.
+
+#### 8.) System shall follow OSHA standard 1910.304-305 for wiring design and protection. [Origin: OSHA standards]
+
+The mounting subsystem should be designed such that wires are easily identified and discerned from other conductors, for example the ground wire must be easily found when comparing it to live conductors. The wires must also be insulated and be in a proper location to minimize the probability of electrical shock occuring.
+
+#### 9.) System shall follow the IEEE 518-1982 Guide for the Installation of Electrical Equipment to Minimize Electrical Noise Inputs to Controllers from External Sources. [Origin: IEEE standards]
+
+The mounting subsystem needs to follow this guide to limit outside noise which could lower the effectiveness of the noise cancellation capability of the overall system.
+
+#### 10.) Overall weight of mounting subsystem cannot exceed 17 lbs. [Origin: Weight requirement ]
+
+The mounting subsystem cannot be physically drilled into the wall since this would damage Tennessee Tech property, therefore command strips will be utilized, these command strips are rated for 20 lbs. The system will add weight, therefore to compensate for the additional weight, the weight requirement will be lessened to a maximum amount of 17 lbs for the physical mount to prevent failure.
+#### 11.) The width of the physical mount cannot exceed 49 in. [Origin: Window width measurements ]
+
+The physical mount cannot be wider than the window in order to be fastened to the window. The measured width of the window in Brown 314 is 49 inches (4 feet 1 inch), therefore the selected physical mount must meet this requirement.
+
+
+
+
+
+
+## Buildable schematic 
```

Comment:
Not buildable. External knowledge is needed. things aren't to scale. It is very ambiguous.